src/components/LogOutBtn.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div v-show="showBtn" class="pa-3 bottom">
+    <v-btn @click.stop="logOut()" flat>
+      <span class="mr-2">salir de la app</span>
+      <v-icon>exit_to_app</v-icon>
+    </v-btn>
+  </div>
+</template>
+<script>
+import { mapActions } from 'vuex'
+
+export default {
+  name: 'LogOutBtn',
+  props: {
+    showBtn: {
+      type: Boolean,
+      default: false
+    }
+  },
+  methods: {
+    ...mapActions(['logout']),
+    logOut () {
+      this.logout().then(route => this.$router.push(route))
+    }
+  }
+}
+</script>
```

Comment:
Do we really need an authentication mechanism to add signatures?

Reply:
That is a mechanism only for administrators, anyone can sign without registration, but I designed a control panel to prevent anyone from going crazy. Register to see the admin panel, you must tell me which email you used to give admin access to the database.

Reply:
Gotcha!